src/test/unit/service/inputFilterService.test.ts

```diff
@@ -1,67 +1,71 @@
 import {InputFilterService} from '../../../main/service/inputFilterService';
 import {expect} from 'chai';
+import fs from "fs";
+import path from "path";
 
 const inputService = new InputFilterService();
 
-const validSearchInputName = 'Aylesbury Magistrate\'s Court';
-const validSearchInputLocation = 'Aylesbury';
-const validSearchInputJurisdiction = 'Magistrates Court';
+const validSearchInputName = 'Accrington Magistrates\' Court';
+const validSearchInputLocation = 'London';
+const validSearchInputJurisdiction = 'Crown Court';
 const validCheckAgainst = ['name', 'location', 'jurisdiction'];
 
 const invalidSearchInputEmpty = '';
 const invalidSearchInputUndefined = undefined;
 
 const expectedResultFromName = [{
-  courtId: 32,
-  name: 'Aylesbury Magistrate\'s Court',
-  jurisdiction: 'Crown Court',
-  location: 'Aylesbury',
-  hearings: 6,
+  "courtId": 3,
+  "name": "Accrington Magistrates' Court",
+  "jurisdiction": "Crown Court",
+  "location": "Manchester"
 }];
 
 const expectedResultFromLocation = [
   {
-    courtId: 34,
-    name: 'Aylesbury Crown Court',
-    jurisdiction: 'Crown Court',
-    location: 'Aylesbury',
-    hearings: 8,
+    "courtId": 2,
+    "name": "Accrington County Court",
+    "jurisdiction": "Crown Court",
+    "location": "London"
   },
   {
-    courtId: 32,
-    name: 'Aylesbury Magistrate\'s Court',
-    jurisdiction: 'Crown Court',
-    location: 'Aylesbury',
-    hearings: 6,
-  }];
+    "courtId": 6,
+    "name": "Alton Magistrates' Court",
+    "jurisdiction": "Royal Court",
+    "location": "London"
+  },
+
+];
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/courtsAllReduced.json'), 'utf-8');
+const courtsData = JSON.parse(rawData);
 
 describe('Input filter service', () => {
   it('should return filtered list with 1 match', () => {
-    expect(inputService.findCourts(validSearchInputName, validCheckAgainst)).to.deep.equal(expectedResultFromName, 'Result did not match expected');
+    expect(inputService.findCourts(validSearchInputName, validCheckAgainst, courtsData)).to.deep.equal(expectedResultFromName, 'Result did not match expected');
```

Comment:
can we break these lines into couple of rows please?
Don't really want to scroll sideways to read whole line